src/useTimer.ts

```diff
@@ -66,12 +62,12 @@ export const useTimer = (config?: IConfig): IValues => {
   };
 
   const start = () => {
-    if (intervalRef.current) {
-      return;
+    if (intervalRef.current === null) {
+      createTimer();
+    }
+    if (endTime > 0) {
```

Comment:
We must check nullity for endTime. endTime could be defined to 0 by user, for example if timer is decremental: initialTime is 50, and endTime is 0.